framework/include/base/NeighborResidualObject.h

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
```

Comment:
This should be merged into previous commit. The commit message is not right.

Reply:
You've never used `git commit --fixup` and `git rebase --autosquash` ? It is easy to autosquash all these commits later. If you give me permission to force push willy nilly to this branch, then I won't push up `fixup` commits anymore

Reply:
Oh, you cannot do force push?

Reply:
My repository is public. I thought anyone can force push to my branches. If not, where should set this up?

Reply:
Oh maybe feel free to create a new PR with your own branch.

Reply:
I can. Was just not doing it at first out of courtesy to you. But now that you've said this branch is all mine, I'll force push :+1: